gym/spec/build_command_generator_spec.rb

```diff
@@ -213,10 +215,35 @@
                                "-archivePath #{Gym::BuildCommandGenerator.archive_path.shellescape}",
                                "OTHER_SWIFT_FLAGS=\"-Xfrontend -debug-time-function-bodies\"",
                                :archive,
-                               "| tee #{log_path.shellescape}",
-                               "| grep .[0-9]ms | grep -v ^0.[0-9]ms | sort -nr > culprits.txt",
+                               "| tee #{@log_path.shellescape}",
+                               "| xcpretty"
```

Comment:
I think this is a duplicate since `xcpretty` gets added here - https://github.com/fastlane/fastlane/pull/12527/files?utf8=✓&diff=unified#diff-440c4122de8c51769dc40db14188f030R70... right? 🤔

Reply:
I think the git diff is misleading here. The `| xcpretty` command is part of the `pipe` method, which does indeed manifest in the [`BuildCommandGenerator.generate`](https://github.com/fastlane/fastlane/blob/master/gym/lib/gym/generators/build_command_generator.rb#L14) output that this test is testing.

I added another test case, which pushed the old `xcpretty` [down a few lines](https://github.com/fastlane/fastlane/pull/12527/files?utf8=%E2%9C%93&diff=unified#diff-8f4a50ba33f7e4e77a65a6a50db78273L218).

Reply:
My bad, that was in a spec 😝 You good 👍